gaming/README.md

```diff
@@ -0,0 +1,153 @@
+# Spanner Gaming Samples
+
+This repository contains sample code for the following use-cases when using Cloud Spanner for the backend:
+
+- Player creation, login, and skin changes
+
+## How to use this
+
+### Setup infrastructure
+A terraform file is provided that creates the appropriate resources for these samples.
+
+Resources that are created:
+- Spanner instance and database based on user variables in main.tfvars
+- (FUTURE) GKE cluster to run the load generators
+
+To set up the infrastructure, do the following:
+
+- Copy `infrastructure/terraform.tfvars.sample` to `infrastructure/terraform.tfvars`
+- Modify `infrastructure/terraform.tfvars` for PROJECT and instance configuration
+- `terraform apply` from within infrastructure directory
+
+```
+cd infrastructure
+cp terraform.tfvars.sample terraform.tfvars
+vi terraform.tfvars # modify variables
+
+terraform apply
+```
+
+### Setup schema
+Schema is managed by [Wrench](https://github.com/cloudspannerecosystem/wrench).
+
+After installing wrench, migrate the schema by running the `schema.bash` file (replace project/instance/database information with what was used in terraform file):
+
+```
+export SPANNER_PROJECT_ID=PROJECTID
+export SPANNER_INSTANCE_ID=INSTANCEID
+export SPANNER_DATABASE_ID=DATABASEID
+./schema.bash
+```
+
+### Player profile sample
+- Configure the `profile-service` by copying the `profile-service/config.yml.template` file to `profile-service/config.yml`, and modify the Spanner connection details:
+```
+spanner:
+  project_id: YOUR_GCP_PROJECT_ID
+  instance_id: YOUR_SPANNER_INSTANCE_ID
+  database_id: YOUR_SPANNER_DATABASE_ID
+
+```
+
+- Run the profile service
+
+```
+cd src/golang/profile-service
+go run .
+```
+
+- Configure the `matchmaking-service` by copying the `matchmaking-service/config.yml.template` file to `matchmaking-service/config.yml`, and modify the Spanner connection details:
+```
+spanner:
+  project_id: YOUR_GCP_PROJECT_ID
+  instance_id: YOUR_SPANNER_INSTANCE_ID
+  database_id: YOUR_SPANNER_DATABASE_ID
+
+```
+
+- Run the match-making service
+
+```
+cd src/golang/matchmaking-service
+go run .
+```
+
```

Comment:
How come there's no mention of the other services (`item`, `profile`, `tradepost`) here?